api/types/types_drivers_executor.go

```diff
@@ -159,7 +182,8 @@ type ProvidesStorageExecutorCLI interface {
 // StorageExecutorCLI provides a way to interact with the CLI tool built with
 // the driver implementations of the StorageExecutor interface.
 type StorageExecutorCLI interface {
-	StorageExecutorWithSupported
+	StorageExecutorFunctions
+	StorageExecutorWithMount
```

Comment:
As discussed on the phone, there are two interfaces:

1. `StorageExecutorWithSupported`
2. `StorageExecutorCLI`

The former is for the platform implementations of the executor and the latter is for the executor CLI itself. They're both required and correct.